src/Aspire.Dashboard/Components/Pages/ConsoleLogs.razor.cs

```diff
@@ -710,6 +733,6 @@ public string GetUrlFromSerializableViewModel(ConsoleLogsPageState serializable)
 
     public ConsoleLogsPageState ConvertViewModelToSerializable()
     {
-        return new ConsoleLogsPageState(PageViewModel.SelectedResource?.Name);
+        return new ConsoleLogsPageState(PageViewModel.SelectedOption.Id is not null ? PageViewModel.SelectedOption.Name : null);
```

Comment:
Add a comment about why you're doing this. Why is id used sometimes and name used used other times?

Reply:
this is to filter out _noSelection

Reply:
added a comment